index.html

```diff
@@ -257,8 +257,19 @@ <h3>
           serves as the <a data-cite=
           "accname-1.2#dfn-accessible-name">accessible name</a> of that
           <a>screenshots object</a>. For accessibility, authors are encouraged
-          to provide a <a>label</a> for each screenshot.
+          to provide a <a>label</a> for each screenshot. User agents SHOULD use
+          this field as the alternative text for the rendered screenshot.
```

Comment:
As we we are not really doing conformance requirements for user agents/browsers, it might be enough to say: 

```suggestion
          to provide a <a>label</a> for each screenshot. 
          This member can serve as alternative text for the rendered screenshot.
```

Reply:
[Chrome recently landed an advanced install dialog that does use this field](https://chromestatus.com/feature/6422599217184768) and Edge has a distinct implementation in progress as well, so is it still the case that we don’t need conformance language?

Reply:
I would argue that we don't. It doesn't itself affect web application's behavior - it's purely a presentational thing that happens independently of the web app itself.